### PR TITLE
Update 14

### PR DESCRIPTION
**Pull request by Rainb0wKey**
#BDFD_WIKI
Fix $isTicket argument description!

### CODE REVIEW

File: src/bdscript/categoryChannels.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+List channel property under given category.
+
+## Syntax
+```
+$categoryChannels[category ID;separator;(option)]
+```
+### Parameters
+- `category ID` `(Type: Snowflake || Flag: Required)`: The category to getting the channels. 
+- `seperator` `(Type: String || Flag: Emptiable)`: The separator that separates each option.
+- `option` `(Type: Enum || Flag: Optional)`: The option to getting channels from the provided `category ID`. _(Default  is `name`)_
```
Comment (MineBartekSA) [inline src/bdscript/categoryChannels.md]:
```suggestion
- `category ID` `(Type: Snowflake || Flag: Required)`: The category from which to take the channels. 
- `seperator` `(Type: String || Flag: Emptiable)`: The separator to use when separating channel properties.
- `option` `(Type: Enum || Flag: Optional)`: Which property to get from category channels. _(Default  is `name`)_
```

File: src/bdscript/categoryChannels.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### Options
+- `name`: The name of the channel.
+- `id`: The id of the channel.
+- `mention`: The channel mention.
```
Comment (MineBartekSA) [inline src/bdscript/categoryChannels.md]:
```suggestion
- `name`: The name of the channels.
- `id`: The id of the channels.
- `mention`: The channel mentions.
```

File: src/bdscript/channelNames.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# $channelNames
+List all channel names separated by given separator.
+
+## Syntax
+```
+$channelNames[separator;(guild ID)]
+```
+### Parameters
+- `seperator` `(Type: String || Flag: Emptiable)`: The separator to separate the channels with.
+- `guild ID` `(Type: Snowflake || Flag: Optional)`: The guild to return it's channels.
```
Comment (MineBartekSA) [inline src/bdscript/channelNames.md]:
```suggestion
- `seperator` `(Type: String || Flag: Emptiable)`: The separator used to separate the channel names.
- `guild ID` `(Type: Snowflake || Flag: Optional)`: The guild for which to return the channel names. _(Defaults to the current guild)_
```

File: src/bdscript/isTicket.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# $isTicket
+Check if the current or given channel is a ticket.
+
+## Syntax
+```
+$isTicket[(channel ID)]
+```
+### Parameters
+- `channel ID` `(Type: Snowflake || Flag: Optional)`: !update it please!
```
Comment (MineBartekSA) [inline src/bdscript/isTicket.md]:
```suggestion
- `channel ID` `(Type: Snowflake || Flag: Optional)`: The channel to check. _(Defaults to the current channel)_
```

File: src/bdscript/isTicket.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+> Use `$isTicket[]` to get the current channel id.
```
Comment (MineBartekSA) [inline src/bdscript/isTicket.md]: I'm sorry, what?

File: src/bdscript/isTicket.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+## Example
+```
+$nomention
+$onlyIf[$isTicket[]==true;This command can only be used in a ticket!]
+This is a ticket!
+```
+![example](https://user-images.githubusercontent.com/113303649/212466758-7a11f906-f356-438d-8f64-0977c28b5a3c.png)
+
+![example](https://user-images.githubusercontent.com/113303649/212466712-3e8d1768-af20-4ea3-9402-8668ae8d9ccc.png)
+> To create a ticket use a [`$newTicket[]`](./newTicket.md) function.
```
Comment (MineBartekSA) [inline src/bdscript/isTicket.md]:
```suggestion
> To create a ticket use the [`$newTicket[]`](./newTicket.md) function.
```

File: src/bdscript/randomCategoryID.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# $randomCategoryID
+Returns random category id from the current server or the provided server.
```
Comment (MineBartekSA) [inline src/bdscript/randomCategoryID.md]:
```suggestion
Returns a random category id from the current server or from the provided server.
```

File: src/bdscript/randomCategoryID.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+## Syntax
+```
+$randomCategoryID[(guild ID)]
+```
+### Parameters
+- `guild ID` `(Type: Snowflake || Flag: Optional)`: The server to get a random catgeory ID.
```
Comment (MineBartekSA) [inline src/bdscript/randomCategoryID.md]:
```suggestion
- `guild ID` `(Type: Snowflake || Flag: Optional)`: The server from which to get a random category ID. _(Defaults to the current server)_
```

File: src/bdscript/randomCategoryID.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+> Use `$randomCategoryID[]` to get the current guild id.
```
Comment (MineBartekSA) [inline src/bdscript/randomCategoryID.md]: ????

File: src/bdscript/randomGuildID.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+# $randomGuildID
+It's returns a random guild ID that the bot is in.
```
Comment (MineBartekSA) [inline src/bdscript/randomGuildID.md]:
```suggestion
Returns a random guild ID from servers the bot is in.
```

File: src/bdscript/randomRoleID.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
+# $randomRoleID
+Returns random role id from the current server or the provided server.
```
Comment (MineBartekSA) [inline src/bdscript/randomRoleID.md]:
```suggestion
Returns a random role id from the current server or from the provided server.
```

File: src/bdscript/randomRoleID.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+## Syntax
+```
+$randomRoleID[(guild ID)]
+```
+### Parameters
+- `guild ID` `(Type: Snowflake || Flag: Optional)`: The server to get a random role id.
+> Use `$randomRoleID[]` to get the current guild id.
```
Comment (MineBartekSA) [inline src/bdscript/randomRoleID.md]: ????????????????????

File: src/bdscript/randomRoleID.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
+# $randomRoleID
+Returns random role id from the current server or the provided server.
+
+## Syntax
+```
+$randomRoleID[(guild ID)]
+```
+### Parameters
+- `guild ID` `(Type: Snowflake || Flag: Optional)`: The server to get a random role id.
```
Comment (MineBartekSA) [inline src/bdscript/randomRoleID.md]:
```suggestion
- `guild ID` `(Type: Snowflake || Flag: Optional)`: The server from which to get a random role id. _(Defaults to the current server)_
```

File: src/bdscript/userBanner.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+# $userBanner
+Returns a user banner if user has one.
```
Comment (MineBartekSA) [inline src/bdscript/userBanner.md]:
```suggestion
Returns user banner if the given user has one.

```

File: src/bdscript/userBanner.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+## Syntax
+```
+$userBanner[user ID]
+```
+ > It doesn't return the user's guild banner.
```
Comment (MineBartekSA) [inline src/bdscript/userBanner.md]:
```suggestion
> The function is unable to return the user's server banner.

```

File: src/bdscript/userBanner.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+### Parameters
+- `user ID` `(Type: Snowflake || Flag: Required)`: The user to get banner for.
```
Comment (MineBartekSA) [inline src/bdscript/userBanner.md]:
```suggestion
- `user ID` `(Type: Snowflake || Flag: Required)`: The user to get the banner for.

```

File: src/bdscript/userBanner.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+## Example
+```
+$nomention
+$sendMessage[$userBanner[$findUser[$message]]?size=4096]
+```
+![example](https://user-images.githubusercontent.com/113303649/212497271-e2ebad00-2d78-4bdf-a697-c2863df527e2.png)
+
+![example](https://user-images.githubusercontent.com/113303649/212497313-3824e855-d634-4e8a-848e-975d6d768df3.png)
+> You can use `?size=size` at the end of the avatar URL to increase/decrease the image size. Example sizes: 1024, 2048, 4096.
```
Comment (MineBartekSA) [inline src/bdscript/userBanner.md]:
```suggestion
> You can use `?size=size` at the end of the banner URL to increase/decrease the image size. Example sizes: 1024, 2048, 4096. 
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

File: src/bdscript/isTicket.md (hunk 5 above)
Comment (Rainb0wKey) [reply src/bdscript/isTicket.md]: ?

Comment (Rainb0wKey) [reply src/bdscript/isTicket.md]: oh

**Comment by Rainb0wKey** [comment]
Fixed^
Sorry

File: src/bdscript/channelNames.md [hunk 18]
```diff
@@ -0,0 +1,18 @@
+# $channelNames
+List all channel names separated by given separator.
```
Comment (MineBartekSA) [inline src/bdscript/channelNames.md]:
```suggestion
List all channel names separated by a given separator.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work!

**Pull request merged**